src/python/espressomd/lb.pyx

```diff
@@ -70,10 +71,21 @@ cdef class HydrodynamicInteraction(Actor):
 
         if self._params["dens"] == default_params["dens"]:
             raise Exception("LB_FLUID density not set")
-        else:
-            if not (self._params["dens"] > 0.0 and (is_valid_type(self._params["dens"], float) or is_valid_type(self._params["dens"], int))):
-                raise ValueError("Density must be one positive double")
-
+        elif not (self._params["dens"] > 0.0 and (is_valid_type(self._params["dens"], float) or is_valid_type(self._params["dens"], int))):
+            raise ValueError("Density must be one positive double")
+        
+        tau = self._params["tau"]
+        if (tau <= 0.0):
+            raise Exception("LB_FLUID tau not set")
+        elif global_time_step > 0:
+            if (tau - global_time_step) / (abs(tau) + abs(global_time_step)) < -numeric_limits[float].epsilon():
```

Comment:
it would probably be cleaner to check that in the core, *e.g.* in ```lb_interface.cpp``` in ```lb_lbfluid_on_lb_params_change```

Reply:
do you want the check of the `time_step` in the core, too? (`mpi_set_time_step()` in `communication.cpp`) ?

Reply:
yes